app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/state/MessageComposeInputState.kt

```diff
@@ -62,7 +62,8 @@ sealed class MessageComposeInputState {
     val isExpanded: Boolean
         get() = this is Active && this.size == MessageComposeInputSize.EXPANDED
     val attachmentOptionsDisplayed: Boolean
-        get() = this is Active && this.type is MessageComposeInputType.NewMessage && this.type.attachmentOptionsDisplayed
+        get() = (this is Active && this.type is MessageComposeInputType.NewMessage && this.type.attachmentOptionsDisplayed)
+                || (this is Active && this.type is MessageComposeInputType.SelfDeletingMessage && this.type.attachmentOptionsDisplayed)
```

Comment:
```suggestion
        get() = (this is Active && this.type.attachmentOptionsDisplayed) &&
    (this.type is MessageComposeInputType.NewMessage || this.type is MessageComposeInputType.SelfDeletingMessage)
```

Just making a bit more ⭐  so we don't have to check for `this is Active` and `this.type.attachmentOptionsDisplayed` twice

Reply:
thanks a lot for suggestion <3 but this will not work :S I tried :S